<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVideoPlayer</class>
 <widget class="QWidget" name="QtVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="sli_moveVideo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>461</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sli_hue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>580</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sli_satur">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bri">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>550</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_satur">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>饱和度</string>
   </property>
  </widget>
  <widget class="QSlider" name="sli_contra">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>550</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_hue">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>色相</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_contra">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>对比度</string>
   </property>
  </widget>
  <widget class="QSlider" name="sli_bri">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
